src/types/pushCleaners.ts

```diff
@@ -27,34 +27,31 @@ export const asBase64 = asCodec(
   clean => base64.stringify(clean)
 )
 
-export const asAddressBalanceTrigger: Cleaner<AddressBalanceTrigger> = asObject(
-  {
-    type: asValue('address-balance'),
-    pluginId: asString,
-    tokenId: asOptional(asString),
-    address: asString,
-    aboveAmount: asOptional(asString), // Satoshis or Wei or such
-    belowAmount: asOptional(asString) // Satoshis or Wei or such
-  }
-)
+export const asAddressBalanceTrigger = asObject<AddressBalanceTrigger>({
+  type: asValue('address-balance'),
+  pluginId: asString,
+  tokenId: asOptional(asString),
+  address: asString,
+  aboveAmount: asOptional(asString), // Satoshis or Wei or such
+  belowAmount: asOptional(asString) // Satoshis or Wei or such
+})
 
-export const asPriceChangeTrigger: Cleaner<PriceChangeTrigger> = asObject({
+export const asPriceChangeTrigger = asObject<PriceChangeTrigger>({
   type: asValue('price-change'),
-  pluginId: asString,
```

Comment:
Looks like the client-side depends on this as part of Matt's work for price notifications.